hail/www/references.md

```diff
@@ -0,0 +1,90 @@
+
+# Scientific Output from the Hail Community  
+
+The following is a sample of research results that have used Hail as some part of the
+scientific pipeline.  If you would like your work added to this list, please
```

Comment:
can this be a link to the "edit this page on github" link?

Reply:
can link directly here:

https://github.com/hail-is/hail/edit/master/hail/www/references.md

Reply:
Yep, that's what I did.  Tried to find a way to make a local link, but didn't seem to be anything.  Squashed the dummy commits.